aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -45,6 +45,8 @@ The following characteristics apply when hosting in-process:
 
 * Client disconnects are detected. The [HttpContext.RequestAborted](xref:Microsoft.AspNetCore.Http.HttpContext.RequestAborted*) cancellation token is cancelled when the client disconnects.
 
+* `Directory.GetCurrentDirectory()` will return the worker directory of the process started by IIS rather than the application directory (ex: C:\Windows\System32\inetsrv for w3wp.exe)
```

Comment:
* Use "returns" over "will return" (present tense)
* Change "ex" to "for example," (with the comma)
* Emphasize the path (asterisks on "C:\Windows\System32\inetsrv" and "w3wp.exe")
* Period at the end

Here's a cut-'n-paste ...

> \* \`Directory.GetCurrentDirectory()\` returns the worker directory of the process started by IIS rather than the application directory (for example, \*C:\Windows\System32\inetsrv\* for \*w3wp.exe\*).